tests/rules/method.js

```diff
@@ -360,7 +360,10 @@ eslintTester.run("method", rule, {
         },
         {
             code: "x.setHTML(evil, { sanitizer: new Sanitizer()})"
-        }
+        },
+        {
+            code: "(info.current = type)(child_ctx)",
+        },
```

Comment:
@mozfreddyb the change looks reasonable to me, I was just wonderingf if it would be worth also adding another test case for something that we expect to be reported as invalid, e.g. something like:

```
{
            code: "(info.current = n.insertAdjacentHTML)('beforebegin', c)",
            errors: [
                {
                    message: /Unsafe call to n.insertAdjacentHTML for argument 1/,
                    type: "CallExpression"
                }
            ],
        },
```

wdyt?

Reply:
Of course, I should have added those kinds of test cases. Thanks for calling it out!

Reply:
I added your suggested test case in another commit and added the allowed case `(info.current = n.insertAdjacentHTML)('beforebegin', 'innocent')`. I don't think this needs any further discussion. Thanks also for the early r+ :)